crypto/algorithms/src/main/java/org/hyperledger/besu/crypto/altbn128/AltBn128Fq12Pairer.java

```diff
@@ -20,8 +20,8 @@
 import java.util.Arrays;
 
 /**
- * Adapted from the pc_ecc (Apache 2 License) implementation:
- * https://github.com/ethereum/py_ecc/blob/master/py_ecc/bn128/bn128_field_elements.py
+ * Adapted from the pc_ecc (Apache 2 License) <a
+ * href="https://github.com/ethereum/py_ecc/blob/master/py_ecc/bn128/bn128_field_elements.py">implementation</a>.
```

Comment:
this is a 404. maybe just https://github.com/ethereum/py_ecc/tree/main/py_ecc/bn128

Reply:
good catch. Let me fix it.